### PR TITLE
#185467894 Bug fix on the page redirection on different users authentication

### PR DESCRIPTION
**Pull request by frankflej**
## What does this PR do?
This is PR mainly is for the page redirection on successfull login
## Description of Task to be completed?
- Correct page redirection on different users authentication
- Styling of the login page and sign up page with its responsiveness
- Hiding the chat icon on the login page
- Making the login logo clickable

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-page-direction-185467894`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

## What are the relevant pivotal tracker/Trello stories?
[#185467894](https://www.pivotaltracker.com/story/show/185467894)

## Add a screenshot?

https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/108126328/b6568ed2-7af7-4f3c-a9cf-728d0a163fdb

### CODE REVIEW

**Review by bergad37 — COMMENTED** [review]
Add short video of how it works!!!

File: src/containers/LoginContainer.jsx [hunk 1]
```diff
@@ -20,25 +20,32 @@ const LoginContainer = () => {
...
     if (isSuccess) {
       // is he a seller
       if (localStorage.getItem('isSeller')) {
         // create popup
         document.querySelector('.overlay').style.display = 'flex';
       } else {
-        navigate('/');
-        window.location.reload();
+        if(localStorage.getItem('isBuyer')){
+          console.log("helloooo")
```
Comment (jcharles921) [inline src/containers/LoginContainer.jsx]: Remove the console

**Review by jcharles921 — APPROVED** [review]

**Review by irakevine — APPROVED** [review]
Great Work!! @frankflej   this fix was needed

**Review by ReneNtabana — APPROVED** [review]

File: src/styles/navbar.scss
Comment (nishimweprince) [inline src/styles/navbar.scss:1]: This style usually affects the visibility of other components below the navbar, take example on the `/product`  or `/category` pages (unless you add a very large margin on the containers). I would suggest to leave out this style, at least for now.

**Review by nishimweprince — CHANGES_REQUESTED** [review]
I left a comment for your review, let me know what you think!

**Review by nishimweprince — APPROVED** [review]

**Pull request merged**